spec/defines/vhost_spec.rb

```diff
@@ -2020,8 +2020,7 @@
             it { is_expected.to compile }
             it { is_expected.to contain_concat('25-rspec.example.com.conf') }
 
-            if (os_facts[:os]['family'] == 'RedHat' && os_facts[:os]['release']['major'].to_i > 6) ||
-               (os_facts[:os]['name'] == 'SLES' && os_facts[:os]['release']['major'].to_i > 11)
+            if os_facts[:os]['family'] == 'RedHat' || os_facts[:os]['name'] == 'SLES'
```

Comment:
Note I said that you probably can remove the whole if and always check for the full bit, including content.

Reply:
Thanks for the suggestion @ekohl , Also I have tried this with removing if condition which usually results in failing parallel specs.
Currently content part has been tested for Redhat and SLES OS, for other OS content part is not checked only concatenated fragment part is checked. So I believe we need this part.